### ISSUE TITLE
installation error

### ISSUE THREAD

**Issue by yarden** [jakobandersen/mod#5]
I'm trying to install mod on Mac OS X. I installed autoconf and related tools using Homebrew:

```
$ brew install autoconf automake libtool
```

and then ran into this error:

```
$ ./bootstrap.sh 
VERSION
src/Makefile.am
doc/Makefile.am
glibtoolize: putting auxiliary files in AC_CONFIG_AUX_DIR, 'auxConfig'.
glibtoolize: copying file 'auxConfig/ltmain.sh'
glibtoolize: putting macros in AC_CONFIG_MACRO_DIRS, 'm4'.
glibtoolize: copying file 'm4/libtool.m4'
glibtoolize: copying file 'm4/ltoptions.m4'
glibtoolize: copying file 'm4/ltsugar.m4'
glibtoolize: copying file 'm4/ltversion.m4'
glibtoolize: copying file 'm4/lt~obsolete.m4'
glibtoolize: Consider adding '-I m4' to ACLOCAL_AMFLAGS in Makefile.am.
configure.ac:87: error: possibly undefined macro: AC_MSG_ERROR
      If this token and others are legitimate, please use m4_pattern_allow.
      See the Autoconf documentation.
configure.ac:121: error: possibly undefined macro: AC_MSG_NOTICE
autoreconf: /usr/local/Cellar/autoconf/2.69/bin/autoconf failed with exit status: 1
```

any ideas on how to fix this? Also, as minor point, the installation instructions refer to ``./repo-bootstrap.sh`` but the script name seems to be ``bootstrap.sh``. Thank you.

**Comment by waldeyr** [comment]
I'm not a Mac user, but as far as I know, brew doesn't install the packages in standard folders.
Maybe, if you install by compiling them individually:

```
curl -OL http://ftpmirror.gnu.org/autoconf/autoconf-2.69.tar.gz
tar -xzf autoconf-2.69.tar.gz 
cd autoconf-2.69
./configure && make && sudo make install
 
curl -OL http://ftpmirror.gnu.org/automake/automake-1.14.tar.gz
tar -xzf automake-1.14.tar.gz
cd automake-1.14
./configure && make && sudo make install
 
curl -OL http://ftpmirror.gnu.org/libtool/libtool-2.4.2.tar.gz
tar -xzf libtool-2.4.2.tar.gz
cd libtool-2.4.2
./configure && make && sudo make install
```

**Comment by jakobandersen** [comment]
Thanks for reporting. I never really found out what was going on, but this particular issue is technically resolved now as v0.9 uses CMake instead of Autotools. Please open a new issue if the it doesn't work with CMake on Mac. Unfortunately I can not test that scenario.
